MvvmCross/Droid/Shared/Presenter/MvxFragmentsPresenter.cs

```diff
@@ -21,8 +21,8 @@ public class MvxFragmentsPresenter
     {
         public const string ViewModelRequestBundleKey = "__mvxViewModelRequest";
 
-        private readonly FragmentHostRegistrationSettings _fragmentHostRegistrationSettings;
-        private readonly Lazy<IMvxNavigationSerializer> _lazyNavigationSerializerFactory;
+        protected FragmentHostRegistrationSettings _fragmentHostRegistrationSettings;
+        protected Lazy<IMvxNavigationSerializer> _lazyNavigationSerializerFactory;
```

Comment:
If they are only assigned in the constructor I would leave them as `private readonly` so that they never get assigned again.  Then expose them as say: `protected FragmentHostRegistrationSettings FragmentHostRegistrationSettings => _fragmentHostRegistrationSettings` so that child classes can still access them.  It just prevents people from changing the reference.